contracts/SpokePool.sol

```diff
@@ -39,21 +43,45 @@ abstract contract SpokePool is Testable, Lockable, MultiCaller {
     // Origin token to destination token routings can be turned on or off.
     mapping(address => mapping(uint256 => bool)) public enabledDepositRoutes;
 
+    struct RelayData {
+        address sender;
+        address recipient;
+        address destinationToken;
+        uint64 realizedLpFeePct;
+        uint64 relayerFeePct;
+        uint64 depositId;
+        uint256 originChainId;
+        uint256 relayAmount;
+    }
+
+    // Each relay is associated with the hash of parameters that uniquely identify the original deposit and a relay
+    // attempt for that deposit. The relay itself is just represented as the amount filled so far. The total amount to 
+    // relay, the fees, and the agents are all parameters included in the hash key.
+    mapping(bytes32 => uint256) relays;
```

Comment:
is `relays` the best name for this variable? what it really is is a mapping of hashes, to their fills, like your comment describes. would a better variable name be `relayFillAmounts` or `relayFills`? curious what you think.

Reply:
relayFills could be good, but it also tracks relays generally right? I think I'll change to `relayFills` unless there's disagreement

Reply:
SGTM!